languages/js/wasm/package.json

```diff
@@ -4,11 +4,13 @@
   "files": [
     "bitwarden_wasm_bg.js",
     "bitwarden_wasm_bg.wasm",
+    "bitwarden_wasm_bg.wasm.d.ts",
+    "bitwarden_wasm_bg.wasm.js",
```

Comment:
This file was missing from here, which causes the package to be published without it. That caused an error when trying to bundle it, as the index.js file tries to import it.

While I was at it, I also added a missing *.d.ts file, and sorted the file list in node/